nwbwidgets/spectrum.py

```diff
@@ -0,0 +1,59 @@
+from ndx_spectrum import Spectrum
+import warnings
+import matplotlib.pyplot as plt
+from collections.abc import Iterable
+from h5py import Dataset
+from ipywidgets import widgets
+import numpy as np
+from ipywidgets.widgets.interaction import show_inline_matplotlib_plots
+
+
+def show_spectrum(node, **kwargs):
+    if isinstance(node, Spectrum):
+        power = _data_to_array(node.power)
```

Comment:
`np.asarray` may work instead of this function

Reply:
done